rpcs3/Emu/Cell/Modules/sceNp.h

```diff
@@ -1121,8 +1121,8 @@ union SceNpSignalingConnectionInfo
 	SceNpId npId;
 	struct
 	{
-		be_t<u32> addr; // in_addr
-		be_t<u16> port; // in_port_t
+		np_in_addr addr; // in_addr
```

Comment:
Why did you remoce be_t<>, seems like a bug.

Reply:
I think there was a reason why I did this but I can't remember it and it doesn't matter for this PR so getting back to be_t.